Extensions/WikiUpdater/WikiUpdaterTask/task/task.json

```diff
@@ -254,7 +254,7 @@
     }
 
 
-        
+
   ],
   "instanceNameFormat": "Git based WIKI File Updater",
   "execution": {
```

Comment:
@rfennell Not sure if this in intentional, but shouldn't line `261` have been updated to `Node10` ? The WikiUpdater still runs on Node6 atm.

Reply:
Nvm, seems to be updated now.

Reply:
@robmaas thanks for spotting that, it was an oversight on my part